src/openrct2-ui/windows/Ride.cpp

```diff
@@ -334,6 +335,7 @@ static Widget _colourWidgets[] = {
     MakeWidget({ 99, 190}, { 12, 12}, WindowWidgetType::ColourBtn, WindowColour::Secondary, 0xFFFFFFFF,          STR_SELECT_ADDITIONAL_COLOUR_1_TIP           ),
     MakeWidget({119, 190}, { 12, 12}, WindowWidgetType::ColourBtn, WindowColour::Secondary, 0xFFFFFFFF,          STR_SELECT_ADDITIONAL_COLOUR_2_TIP           ),
     MakeWidget({100,  74}, {239, 12}, WindowWidgetType::Checkbox,  WindowColour::Secondary, STR_RANDOM_COLOUR                                                 ),
+    MakeWidget({139, 190}, { 80, 12}, WindowWidgetType::Button,    WindowColour::Secondary, STR_RANDOMISE_VEHICLE_COLOURS, STR_RANDOMISE_VEHICLE_COLOURS_TIP  ),
```

Comment:
I think we should consider widening the button a little. It's a snug fit for English, but I think e.g. Dutch and German will require some more space. Perhaps increase to 140px width?

Reply:
You're right; I haven't thought about that.
How does this look?
![image](https://github.com/OpenRCT2/OpenRCT2/assets/104749673/74a385b9-f722-45a1-afbd-77da1d0e5c4a)

Reply:
Hmm, that doesn't look great for English, does it... Perhaps we should go somewhere in between, then? 😕 Not 80px or 140px, but say, 110px?

Reply:
![image](https://github.com/OpenRCT2/OpenRCT2/assets/104749673/d2e543ee-1f11-4a8e-bd3b-c00d1da8e303)
Looks alright to me at 110px 🧐

Reply:
Yeah, that looks like a good compromise. I actually like the extra horizontal space for English, too. Thanks.